seed/python-sdk/pagination/src/seed/users/client.py

```diff
@@ -116,7 +132,7 @@ def list_with_offset_pagination(
         order: typing.Optional[Order] = None,
         starting_after: typing.Optional[str] = None,
         request_options: typing.Optional[RequestOptions] = None,
-    ) -> ListUsersPaginationResponse:
+    ) -> SyncPaginator[User]:
```

Comment:
i don't see `snippets.json` updated for this endpoint, but would be nice to include the `for ... if`

Reply:
It might be easier to include in a different PR